exercises/matrix/example.exs

```diff
@@ -0,0 +1,63 @@
+defmodule MatrixStruct do
```

Comment:
What would you think about calling this module just `Matrix`? I'm cool with assuming that we'll use a struct, but the naming of the module might be a little confusing I think.